npm/webpack-batteries-included-preprocessor/index.js

```diff
@@ -97,28 +97,6 @@ const addTypeScriptConfig = (file, options) => {
   return options
 }
 
-/**
- * Config yarn pnp plugin for webpack 4
- * @param {*} file file to be processed
- * @param {*} options
- */
-const addYarnPnpConfig = (file, options) => {
```

Comment:
no longer needed with webpack 5. This was an oversight from the upgrade from webpack 4 to 5